dj/sql/parsing/ast.py

```diff
@@ -1030,6 +1030,20 @@ def __str__(self) -> str:
         return prefix + str(self.name)
 
 
+@dataclass(eq=False)
+class MapSubscript(Expression):
+    """
+    Map accessors
+    """
+
+    map_column: "Column"
+    keys: List[str]
```

Comment:
Would it be possible for this to be `List[Name]`? That way we can use the `quote_style` attribute on the [Name](https://github.com/DataJunction/dj/blob/main/dj/sql/parsing/ast.py#L469) class to preserve the type of quotes that were originally used when we re-render the SQL.

Reply:
Makes sense. Just added this change.